tests/performance/collect_measurements

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+for i in tests/performance/*/output tests/performance/*/mpirun*/output
+do
+  if [[ "$(wc -l $i 2> /dev/null | awk '{ print $1 }')" -ne 3 ]]; then
+    continue
+  fi
+
+  test_name=${i##tests/performance/}
+  test_name=${test_name%%/output}
+  test_name=${test_name/.release/}
+  test_name=${test_name/.debug/}
```

Comment:
You'll probably want a comment to explain what this does :-)

Reply:
The script file is now properly refactored with commented code.